react/features/toolbox/components/web/Toolbox.js

```diff
@@ -273,6 +289,8 @@ class Toolbox extends Component<Props> {
                     shortcut.helpDescription);
             }
         });
+
+        window.addEventListener('resize', this._onResize);
```

Comment:
Not worth debouncing the callback? Probably an infrequent enough case not to but checking in anyway.

Poking around the mouseover in this file needs debouncing too.

Reply:
Yes. I don't think it worth. Also when I tested it all looked fine...